worksModal.js

```diff
@@ -0,0 +1,183 @@
+    "Project6 text Ipsuz is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  ],
+  projectImgMobile:
+  [
+    'assets/images/p1Mobile.png',
+    'assets/images/p2Mobile.png',
+    'assets/images/p1Mobile.png',
+    'assets/images/p2Mobile.png',
+    'assets/images/p1Mobile.png',
+    'assets/images/p2Mobile.png',
+  ],
+  projectImgDesktop:
+  [
+    'assets/images/p1Desktop.png',
+    'assets/images/p2Desktop.png',
+    'assets/images/p1Desktop.png',
+    'assets/images/p2Desktop.png',
+    'assets/images/p1Desktop.png',
+    'assets/images/p2Desktop.png',
+
+  ],
+  linkLive:
+  [
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    '#',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+  ],
+  linkSource:
+  [
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    '#',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+  ],
+};
+
+const worksGrid = document.getElementById('worksGrid');
+
+for (let i = 0; i < worksInfo.title.length; i += 1) {
+  const article = document.createElement('article');
+  article.className = 'workPost borderRad12 dFlex';
+  const divCard = document.createElement('div');
+  divCard.className = 'workInfo borderRad12 dFlex';
+  const hgroup = document.createElement('hgroup');
+  hgroup.className = 'postColor';
+
+  const title = document.createElement('h2');
+  const textTitle = document.createTextNode(worksInfo.title[i]);
+  title.className = 'cardTitle';
+  title.appendChild(textTitle);
+
+  const techList = document.createElement('ul');
+  techList.className = 'postColor dFlex';
+
+  const x = worksInfo.tech[i];
+
+  for (let j = 0; j < x.length; j += 1) {
+    const li = document.createElement('li');
+    li.innerText = x[j];
+    techList.appendChild(li);
+  }
+
+  const button = document.createElement('a');
+  button.innerText = 'See Project';
+  button.className = 'button';
+  button.href = '#';
+  hgroup.appendChild(title);
+  divCard.appendChild(hgroup);
+  divCard.appendChild(techList);
+  divCard.appendChild(button);
+  article.appendChild(divCard);
+  worksGrid.appendChild(article);
+}
+
+/* create modal window */
+const modal = document.getElementById('modal');
```

Comment:
- [ ] As the previous code reviewer suggested, please try to make your modal fit on the screen and don't let the screen size cut the model off so the user can see the `X` button to close it. I get it that on some screen sizes the model fits perfectly, but on smaller ones like the Iphone SE it doesn't. Please try to make your modal responsive and fits on every kind of screen. (maybe reducing the description could be a good idea)
 -Here is your app on an Iphone SE:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/61919617/188866616-9eb06b4b-74cf-469b-ae7a-3aa7748e057e.png">

Reply:
@leonardodiasb
 thank you for the suggestion. I thought on erasing the text description, but was afraid it would be ask for a review because it wouldn't be exactly like figma (they use a long mobile example in where everything fits)